SpringKonstant/src/org/usfirst/frc6357/robotcode/Robot.java

```diff
@@ -73,7 +73,9 @@ public void robotInit()
         chooserBox.addDefault("Don't pick up a box", "N");
         chooserBox.addObject("Pick up a box", "Y");
         chooserBox.addObject("Test pickup", "T");
-        
+
+        distanceLIDAR = TFMini.getInstance();
+
```

Comment:
This is a required change so we can get a reference to the TFMini singleton object